### PR TITLE
Emoji (symbols) emotion and sentiment text are not aligned on sentences. 

### PR DESCRIPTION
**Pull request by Tehreem-personal**
Fixed the issues stated: Emoji (symbols) emotion and sentiment text are not aligned on sentences. Moreover,
- Sentiments and pie chart content overflow on some browser scale.
- Also "Go back" button is now aligned with preview heading
- Closed eda_83 and eda_99

### CODE REVIEW

File: src/containers/result/styledtext.tsx [hunk 1]
```diff
@@ -24,8 +24,14 @@ export const StyledList = styled(List)`
...
 `;
 export const SentenceText = styled.div`
-  width: ${(props: any) => (props.isSmallScreen ? "250px" : "350px")};
+  width: ${(props: any) => (props.isSmallScreen ? "250px" : "500px")};
   text-align: justify;
+  margin-right: 10%;
+  @media (max-width: 1200px){
+    padding: 1rem;
+   margin-right: 0
+    // justify-content: left;
```
Comment (xiarahman) [inline src/containers/result/styledtext.tsx:33]: remove

File: src/containers/result/colorHelper.tsx [hunk 2]
```diff
@@ -17,13 +17,13 @@ import { LikeOutlined, DislikeOutlined } from "@ant-design/icons";
   export const getEmotionTag = (emotion: string): React.ReactNode => {
```
Comment (xiarahman) [inline src/containers/result/colorHelper.tsx:17]: Kindly comment :(

File: src/containers/result/styledtext.tsx [hunk 3]
```diff
@@ -24,8 +24,14 @@ export const StyledList = styled(List)`
   border-radius: 8px;
 `;
 export const SentenceText = styled.div`
```
Comment (xiarahman) [inline src/containers/result/styledtext.tsx:26]: Kindly comment :((

File: src/containers/result/styledtext.tsx [hunk 4]
```diff
@@ -24,8 +24,14 @@ export const StyledList = styled(List)`
...
   text-align: justify;
+  margin-right: 10%;
+  @media (max-width: 1200px){
+    padding: 1rem;
+   margin-right: 0
+    // justify-content: left;
+  }
 `;
 
 export const LoadMoreButtonWrapper = styled.div`
```
Comment (xiarahman) [inline src/containers/result/styledtext.tsx:37]: Kindly comment 🥇

File: src/containers/result/styledtext.tsx [hunk 5]
```diff
@@ -79,7 +85,15 @@ export const LoadMoreButton = styled(Button)`
 export const GoBackButtonFlex = styled(Flex)`
   min-width: 10px;
   cursor: pointer;
-  margin: 3%;
-  margin-left: 9%;
+  margin: 4%;
+  
   justify-content: start;
 `;
+export const StyledTag = styled.div`
```
Comment (xiarahman) [inline src/containers/result/styledtext.tsx:92]: Kindly comment :@

File: src/containers/result/styledtext.tsx [hunk 6]
```diff
@@ -79,7 +85,15 @@ export const LoadMoreButton = styled(Button)`
 export const GoBackButtonFlex = styled(Flex)`
```
Comment (xiarahman) [inline src/containers/result/styledtext.tsx:85]:
```suggestion
// Button to go back
export const GoBackButtonFlex = styled(Flex)` 
```

**Review by xiarahman — COMMENTED** [review]
Looks fine. Kindly comment all the consts so it is clear what is their function. Good job.

File: src/containers/result/styledtext.tsx (hunk 1 above)
Comment (Tehreem-personal) [reply src/containers/result/styledtext.tsx:33]: removed

File: src/containers/result/colorHelper.tsx [hunk 7]
```diff
@@ -4,11 +4,11 @@ import { LikeOutlined, DislikeOutlined } from "@ant-design/icons";
  export const getSentimentIcon = (sentiment: string): React.ReactNode => {
```
[the remark this answers is not in the document]
Comment (Tehreem-personal) [reply src/containers/result/colorHelper.tsx:4]: added comments in this file

File: src/containers/result/colorHelper.tsx (hunk 2 above)
Comment (Tehreem-personal) [reply src/containers/result/colorHelper.tsx:17]: added

File: src/containers/result/styledtext.tsx [hunk 8]
```diff
@@ -37,7 +43,7 @@ export const LoadMoreButtonWrapper = styled.div`
 export const StyledFlex = styled(Flex)`
```
[the remark this answers is not in the document]
Comment (Tehreem-personal) [reply src/containers/result/styledtext.tsx:43]: Comments added in latest code

File: src/containers/result/styledtext.tsx (hunk 5 above)
Comment (Tehreem-personal) [reply src/containers/result/styledtext.tsx:91]: Comments added in latest code

File: src/containers/result/styledtext.tsx (hunk 3 above)
Comment (Tehreem-personal) [reply src/containers/result/styledtext.tsx:26]: Comments added in latest code

File: src/containers/result/styledtext.tsx (hunk 4 above)
Comment (Tehreem-personal) [reply src/containers/result/styledtext.tsx:36]: Done

**Pull request merged**